documents/charter.md

```diff
@@ -16,6 +16,13 @@ Application providers are even designing algorithms to detect the presence of su
 Instead, it would be beneficial, for both the application provider and network operator, to signal network attributes to the application to self-adapt its video traffic to conform to the specified characteristics.
 The application provider has the ability to measure end user QoE and therefore can self-adapt with QoE feedback.
 
+L4S is a recently standardized mechanism where nodes in the network assist endpoints in discovering optimal sending rates through continuous ECN feedback, interpreted by purpose-built congestion control algorithms. 
+However, L4S alone is not sufficient to communicate network properties related to video self-adaptation. 
+L4S and ECN operate on RTT timescales, reflecting the current state of a network buffer, whereas the network properties related to video traffic reflect policies that tend to be stable over orders of magnitude longer timescales. 
```

Comment:
```suggestion
L4S and ECN operate on RTT timescales, reflecting the current state of a network buffer, whereas the network properties that allow video traffic adaptation reflect policies that tend to be stable over orders of magnitude longer timescales. 
```

Reply:
I think I prefer what Marcus wrote than this suggestion, Spencer.  Still, it could be more brief.

Also

```suggestion
L4S and ECN operate on RTT timescales, whereas network policies tend to be stable over much longer timescales.
```

Reply:
I like Martins proposal. It's brief and straight to the point.